uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyMerged.java

```diff
@@ -305,10 +306,10 @@ protected static boolean isCollapsedCellMixedValue(final GridData model,
     }
 
     protected static double getCellHeight(final int rowIndex,
-                                          final GridData model,
+                                          final List<Double> allRowHeights,
                                           final GridCell<?> cell) {
         return IntStream.range(rowIndex, rowIndex + cell.getMergedCellCount())
```

Comment:
@manstis 
could you please replace with for-loop (or parallel stream) for performance ?

Reply:
Done.